metrics/prometheus.go

```diff
@@ -533,10 +541,34 @@ func (c *PrometheusCollector) Collect(ch chan<- prometheus.Metric) {
 }
 
 const (
-	containerLabelPrefix = "container_label_"
-	containerEnvPrefix   = "container_env_"
+	// ContainerLabelPrefix is the prefix added to all container labels added
+	// to exported metrics.
+	ContainerLabelPrefix = "container_label_"
+	// ContainerEnvPrefix is the prefix added to all env variables added to
+	// exported metrics.
+	ContainerEnvPrefix = "container_env_"
 )
 
+// DefaultContainerLabels implements ContainerLabelsFunc. It exports the
+// container name, first alias, image name as well as all its env and label
+// values.
+func DefaultContainerLabels(container *info.ContainerInfo) map[string]string {
+	set := map[string]string{"id": container.Name}
+	if len(container.Aliases) > 0 {
+		set["name"] = container.Aliases[0]
+	}
+	if image := container.Spec.Image; len(image) > 0 {
+		set["image"] = image
+	}
```

Comment:
> I could split up the id+name+image part and the labels+env parts into two functions we expose each? I'm also fine with exposing constants for the labels though. 

Thinking about this some more, I think just exposing the constants is the way to go.

> Lastly, actually we're doing it all wrong here by applying so many labels to all metrics. It should be only one identifier (probably id) and export another metric containing the mapping. See http://www.robustperception.io/target-labels-are-for-life-not-just-for-christmas/ and http://www.robustperception.io/exposing-the-software-version-to-prometheus/.

Ack. I think this is too big a change for this late in the release though. Could you open an issue with a proposal to fix it, and we can try to address it in the next (v0.25) release. Thanks!